test/sqllogictest/range.slt

```diff
@@ -839,2343 +839,4367 @@ SELECT a::text t, array_agg(v::text ORDER BY v) FROM (
+SELECT ('(,-1)'::int4range * 'empty'::int4range)::text
 ----
-[1,2)
+empty
 
 query T
-SELECT '(1,)'::int8range::text;
+SELECT ('(,-1)'::int4range * '[-2,2)'::int4range)::text
 ----
-[2,)
+[-2,-1)
 
-query error range lower bound must be less than or equal to range upper bound
-SELECT '(1,-1)'::int8range::text;
+query T
+SELECT ('(,-1)'::int4range * '[1,)'::int4range)::text
+----
+empty
 
-query error range lower bound must be less than or equal to range upper bound
-SELECT '(1,0)'::int8range::text;
+query T
+SELECT ('(,-1)'::int4range * '(,)'::int4range)::text
+----
+(,-1)
 
 query T
-SELECT '(1,1)'::int8range::text;
+SELECT ('[-2,2)'::int4range * 'empty'::int4range)::text
 ----
 empty
 
 query T
-SELECT '(1,]'::int8range::text;
+SELECT ('[-2,2)'::int4range * '(,-1)'::int4range)::text
 ----
-[2,)
+[-2,-1)
 
-query error range lower bound must be less than or equal to range upper bound
-SELECT '(1,-1]'::int8range::text;
+query T
+SELECT ('[-2,2)'::int4range * '[1,)'::int4range)::text
+----
+[1,2)
 
-query error range lower bound must be less than or equal to range upper bound
-SELECT '(1,0]'::int8range::text;
+query T
+SELECT ('[-2,2)'::int4range * '(,)'::int4range)::text
+----
+[-2,2)
 
 query T
-SELECT '(1,1]'::int8range::text;
+SELECT ('[1,)'::int4range * 'empty'::int4range)::text
 ----
 empty
 
 query T
-SELECT '[,)'::int8range::text;
+SELECT ('[1,)'::int4range * '(,-1)'::int4range)::text
 ----
-(,)
+empty
 
 query T
-SELECT '[,-1)'::int8range::text;
+SELECT ('[1,)'::int4range * '[-2,2)'::int4range)::text
 ----
-(,-1)
+[1,2)
 
 query T
-SELECT '[,0)'::int8range::text;
+SELECT ('[1,)'::int4range * '(,)'::int4range)::text
 ----
-(,0)
+[1,)
 
 query T
-SELECT '[,1)'::int8range::text;
+SELECT ('(,)'::int4range * 'empty'::int4range)::text
 ----
-(,1)
+empty
 
 query T
-SELECT '[,]'::int8range::text;
+SELECT ('(,)'::int4range * '(,-1)'::int4range)::text
 ----
-(,)
+(,-1)
 
 query T
-SELECT '[,-1]'::int8range::text;
+SELECT ('(,)'::int4range * '[-2,2)'::int4range)::text
 ----
-(,0)
+[-2,2)
 
 query T
-SELECT '[,0]'::int8range::text;
+SELECT ('(,)'::int4range * '[1,)'::int4range)::text
 ----
-(,1)
+[1,)
+
+#
+# int4range difference
 
 query T
-SELECT '[,1]'::int8range::text;
+SELECT (null - 'empty'::int4range)::text
 ----
-(,2)
+NULL
 
 query T
-SELECT '[-1,)'::int8range::text;
+SELECT ('empty'::int4range - 'empty'::int4range)::text
 ----
-[-1,)
+empty
 
 query T
-SELECT '[-1,-1)'::int8range::text;
+SELECT ('empty'::int4range - 'empty'::int4range)::text
 ----
 empty
 
 query T
-SELECT '[-1,0)'::int8range::text;
+SELECT ('empty'::int4range - '(,-1)'::int4range)::text
 ----
-[-1,0)
+empty
 
 query T
-SELECT '[-1,1)'::int8range::text;
+SELECT ('(,-1)'::int4range - 'empty'::int4range)::text
 ----
-[-1,1)
+(,-1)
 
 query T
-SELECT '[-1,]'::int8range::text;
+SELECT ('(,-1)'::int4range - '(,-1)'::int4range)::text
 ----
-[-1,)
+empty
 
 query T
-SELECT '[-1,-1]'::int8range::text;
+SELECT ('empty'::int4range - '(,-1)'::int4range)::text
```

Comment:
Also exists in line 1123, is that intentional?

Reply:
Removed inadvertent duplicates, including this.